src/store/ducks/generic/jurisdictions.ts

```diff
@@ -184,7 +184,7 @@ export type ImmutableGenericJurisdictionState = FullGenericJurisdictionState &
   SeamlessImmutable.ImmutableObject<FullGenericJurisdictionState>;
 
 /** initial GenericJurisdiction state */
-const initialState: ImmutableGenericJurisdictionState = SeamlessImmutable({});
+const initialState: any = SeamlessImmutable({});
```

Comment:
Why any?

Reply:
The app faile dot start. I cherry picked https://github.com/onaio/reveal-frontend/commit/2b75e40a03b4b31f5825a6ac5b13f9ad6bbaf891

Reply:
@kelvin-muchiri This has been resolved in master, well at least in a tad better way than using any.

Reply:
@p-netm @kelvin-muchiri can we then clean up this PR to match what is in master?